[2023-03-15T17:13:04.636+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 01_dag_test.read_from_db manual__2023-03-15T17:13:02.469163+00:00 [queued]>
[2023-03-15T17:13:04.643+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 01_dag_test.read_from_db manual__2023-03-15T17:13:02.469163+00:00 [queued]>
[2023-03-15T17:13:04.645+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-03-15T17:13:04.645+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-03-15T17:13:04.646+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-03-15T17:13:04.656+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): read_from_db> on 2023-03-15 17:13:02.469163+00:00
[2023-03-15T17:13:04.662+0000] {standard_task_runner.py:55} INFO - Started process 4728 to run task
[2023-03-15T17:13:04.666+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', '01_dag_test', 'read_from_db', 'manual__2023-03-15T17:13:02.469163+00:00', '--job-id', '1126', '--raw', '--subdir', 'DAGS_FOLDER/dag-test.py', '--cfg-path', '/tmp/tmpg1cwkv5y']
[2023-03-15T17:13:04.669+0000] {standard_task_runner.py:83} INFO - Job 1126: Subtask read_from_db
[2023-03-15T17:13:04.744+0000] {task_command.py:376} INFO - Running <TaskInstance: 01_dag_test.read_from_db manual__2023-03-15T17:13:02.469163+00:00 [running]> on host f11e2626d663
[2023-03-15T17:13:04.810+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=01_dag_test
AIRFLOW_CTX_TASK_ID=read_from_db
AIRFLOW_CTX_EXECUTION_DATE=2023-03-15T17:13:02.469163+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-15T17:13:02.469163+00:00
[2023-03-15T17:13:04.819+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-03-15T17:13:04.827+0000] {sql.py:315} INFO - Running statement: SELECT * FROM pg_catalog.pg_tables;, parameters: None
[2023-03-15T17:13:04.839+0000] {sql.py:324} INFO - Rows affected: 69
[2023-03-15T17:13:04.852+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=01_dag_test, task_id=read_from_db, execution_date=20230315T171302, start_date=20230315T171304, end_date=20230315T171304
[2023-03-15T17:13:04.920+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-03-15T17:13:04.936+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
